### DISCUSSION TITLE
Feature request: pause remote video until everyone is loaded it

### DISCUSSION THREAD

**Discussion by soredake** [Syncplay/syncplay#538] [Archive]
Hi, i would like to request a feature to pause remote video (youtube for example) until everyone is loaded it, it would be better then let it play, as it's not always resets to start when someone loads it, and i've encountered weird behavior (syncplay is changing time forward and backwards randomly) after a friend loaded a video that i watched first for 20 seconds.

**Comment by Et0h** [comment]
As far as I know the media players don't report if it's loaded so it's not a feature we could provide. However if you're watching YouTube our advice is to watch with mpv (or mpv.net) alongside ytdl as it's much more reliable than VLC in terms of Syncplay compatibility.

You can also use the readiness feature and people can wait until it's fully loaded before setting themselves to ready.

**Comment by soredake** [comment]
@Et0h if i understand this correctly, syncplay knows if current video in playlist is loaded in player (i use mpv), if current video in playlist is not loaded, syncplay marks time and link with red color and underline:
![Снимок экрана 2022-06-05 143138](https://user-images.githubusercontent.com/5204968/172048509-eadaf3bf-9a2a-4247-96f0-b86b10e29fb6.png)

**Comment by soredake** [comment]
>people can wait until it's fully loaded before setting themselves to ready.

Marking yourself as "not ready" while loading every video and marking as ready after loading is really tiring, i think syncplay can just pause video if not everyone is loaded it yet.

**Comment by soredake** [comment]
@Et0h can you reopen this please? I think it's a valid request that can be implemented.

**Comment by Et0h** [comment]
I'll mark it as a feature request in the hope that someone will one day find and contribute a working implementation.

**Reply by soredake** [reply]
Yeah, i'm fine with it, thanks.

**Comment by Et0h** [comment]
This discussion is being locked because nobody has volunteered to take this matter forwards. If someone wishes to develop this feature then please let us know.
